composer.json

```diff
@@ -1,55 +1,60 @@
 {
-    "name": "alleyinteractive/wp-theme-migrator",
-    "description": "A WordPress plugin to migrate to a new theme incrementally.",
-    "type": "wordpress-plugin",
-    "keywords": [
-        "alleyinteractive",
-        "wp-theme-migrator"
-    ],
-    "homepage": "https://github.com/alleyinteractive/wp-theme-migrator",
-    "license": "GPL-2.0-or-later",
-    "authors": [
-        {
-            "name": "Alley",
-            "email": "info@alley.com"
-        }
-    ],
-    "require": {
-        "php": "^8.0",
-        "alleyinteractive/composer-wordpress-autoloader": "^1.0"
+  "name": "alleyinteractive/wp-theme-migrator",
+  "description": "A library to migrate to a new WordPress theme incrementally.",
+  "type": "library",
+  "keywords": [
+    "alleyinteractive",
+    "wp-theme-migrator"
+  ],
+  "homepage": "https://github.com/alleyinteractive/wp-theme-migrator",
+  "license": "GPL-2.0-or-later",
+  "authors": [
+    {
+      "name": "Alley",
+      "email": "info@alley.com"
     },
-    "require-dev": {
-        "alleyinteractive/alley-coding-standards": "^1.0",
-        "mantle-framework/testkit": "^0.10",
-        "nunomaduro/collision": "^5.0"
-    },
-    "config": {
-        "allow-plugins": {
-            "alleyinteractive/composer-wordpress-autoloader": true,
-            "dealerdirect/phpcodesniffer-composer-installer": true,
-            "pestphp/pest-plugin": true
-        },
-        "sort-packages": true
-    },
-    "extra": {
-        "wordpress-autoloader": {
-            "autoload": {
-                "WP_Theme_Migrator\\": "src"
-            },
-            "autoload-dev": {
-                "WP_Theme_Migrator\\Tests\\": "tests"
-            }
-        }
+    {
+      "name": "Contributors",
```

Comment:
@kevinfodness  I hear that. There are aspects of this approach that strike me as more equitable. It doesn't put the onus on anyone to add themselves to a document in order to be recognized. It also acknowledges relative contributions. Unfortunately, it doesn't recognize non-code contributors so there's that. I plan to start an RFC to discuss so we can standardize how we documents contributors.

Reply:
Great idea re: RFC.

Alternately, we could include the link to all contributors in the CONTRIBUTORS.md file. I think this field results in who shows up under "Maintainers" on Packagist, and since this isn't a person or an organization, I think it would be confusing as an entry there. (See, for example, https://packagist.org/packages/alleyinteractive/wp-concurrent-remote-requests, which has Sean as the author instead of Alley.)